### ISSUE TITLE
Error installing stable-2.5 on OSX Sierra 10.12.1

### ISSUE THREAD

**Issue by VictorTaelin** [agda/agda#3017]
I've attempted to install Agda 2.5.4 on OSX Sierra by cloning this repository, checking out the branch `2.5-stable` and following the install instructions. I've installed all dependencies successfully:

```bash
alex -~/vic/dev/agda/agda: alex --version
Alex version 3.2.3, (c) 2003 Chris Dornan and Simon Marlow

~/vic/dev/agda/agda: happy --version
Happy Version 1.19.9 Copyright (c) 1993-1996 Andy Gill, Simon Marlow (c) 1997-2005 Simon Marlow

Happy is a Yacc for Haskell, and comes with ABSOLUTELY NO WARRANTY.
This program is free software; you can redistribute it and/or modify
it under the terms given in the file 'LICENSE' distributed with
the Happy sources.

~/vic/dev/agda/agda: cpphs --version
cpphs 1.20.8

~/vic/dev/agda/agda: cabal --version
cabal-install version 2.2.0.0
compiled using version 2.2.0.0 of the Cabal library

~/vic/dev/agda/agda: ghc --version
The Glorious Glasgow Haskell Compilation System, version 8.4.1
```

I've then went to Agda's root directory and used `make install`. This is the output:

```bash
~/vic/dev/agda/agda: make install
cabal install --disable-documentation --builddir=./dist-2.5.4 --enable-tests --disable-library-profiling --program-suffix=-2.5.4 -fenable-cluster-counting
Resolving dependencies...
Downloading ansi-wl-pprint-0.6.8.2...
Downloading data-default-0.7.1.1...
Configuring blaze-html-0.9.0.1...
Building blaze-html-0.9.0.1...
Downloading filemanip-0.3.6.3...
Configuring data-default-0.7.1.1...
Configuring ansi-wl-pprint-0.6.8.2...
Configuring filemanip-0.3.6.3...
Building data-default-0.7.1.1...
Building ansi-wl-pprint-0.6.8.2...
Building filemanip-0.3.6.3...
Installed data-default-0.7.1.1
Downloading regex-tdfa-text-1.0.0.3...
Configuring regex-tdfa-text-1.0.0.3...
Installed ansi-wl-pprint-0.6.8.2
Downloading temporary-1.2.1.1...
Configuring temporary-1.2.1.1...
Building regex-tdfa-text-1.0.0.3...
Building temporary-1.2.1.1...
Installed filemanip-0.3.6.3
Configuring text-icu-0.7.0.1...
Installed temporary-1.2.1.1
Configuring unordered-containers-0.2.9.0...
Installed regex-tdfa-text-1.0.0.3
Configuring utf8-string-1.0.1.1...
Failed to install text-icu-0.7.0.1
Build log ( /Users/v/.cabal/logs/ghc-8.4.1/text-icu-0.7.0.1-ECUqzqQgfEZE7EoZ5SUFIH.log ):
cabal: Entering directory '/var/folders/rt/rjqmvxfx27q1w_lw1x6hjl7m0000gn/T/cabal-tmp-40676/text-icu-0.7.0.1'
Configuring text-icu-0.7.0.1...
cabal: Missing dependencies on foreign libraries:
* Missing (or bad) C libraries: icuuc, icui18n, icudata
This problem can usually be solved by installing the system packages that
provide these libraries (you may need the "-dev" versions). If the libraries
are already installed but in a non-standard location then you can use the
flags --extra-include-dirs= and --extra-lib-dirs= to specify where they are.If
the library files do exist, it may contain errors that are caught by the C
compiler at the preprocessing stage. In this case you can re-run configure
with the verbosity flag -v3 to see the error messages.

cabal: Leaving directory '/var/folders/rt/rjqmvxfx27q1w_lw1x6hjl7m0000gn/T/cabal-tmp-40676/text-icu-0.7.0.1'
Configuring vector-0.12.0.1...
Building unordered-containers-0.2.9.0...
Building utf8-string-1.0.1.1...
Building vector-0.12.0.1...
Installed utf8-string-1.0.1.1
Installed blaze-html-0.9.0.1
Installed unordered-containers-0.2.9.0
Installed vector-0.12.0.1
cabal: Error: some packages failed to install:
Agda-2.5.4-Re2ohacMmg51kUyHXsroW depends on Agda-2.5.4 which failed to
install.
text-icu-0.7.0.1-ECUqzqQgfEZE7EoZ5SUFIH failed during the configure step. The
exception was:
ExitFailure 1

make: *** [install-bin] Error 1
```

Is "text-icu" some undocumented dependency?

**Comment by gallais** [comment]
Cf #2449

**Comment by VictorTaelin** [comment]
So, in short, this helped:

```
brew install icu4c

cabal install text-icu --extra-lib-dirs=/usr/local/opt/icu4c/lib --extra-include-dirs=/usr/local/opt/icu4c/include

make install CABAL_OPTS='--extra-lib-dirs=/usr/local/opt/icu4c/lib --extra-include-dirs=/usr/local/opt/icu4c/include'
```

(Should probably be on install instructions.)

**Comment by asr** [comment]
Duplicate of #2939.
